src/components/HyperlinkingWrapper.js

```diff
@@ -77,6 +77,14 @@ export default class HyperlinkingWrapper extends Component {
 
                 this.setState({ currentSelection });
                 this.quill.format('highlight', true);
+            } else {
+                const [blot, blotRange] = this.getBlotFromIndex(range.index);
+                const format = blot.formats() || {};
```

Comment:
How about
```
const format = blot.formats() || {};
this.setState({ linkValue: format.link });
```
?

Reply:
done with `this.setState({ selectionFormat });`